File: sample/src/androidTest/java/com/emarsys/sample/testutils/InstrumentedApplication.java
package com.emarsys.sample.testutils;

import android.content.Context;

import com.emarsys.sample.SampleApplication;

import androidx.multidex.MultiDex;

public class InstrumentedApplication extends SampleApplication {

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }

    @Override
    public void onCreate() {
        super.onCreate();
    }
}

File: sample/src/androidTest/java/com/emarsys/sample/testutils/InstrumentedApplication.kt
package com.emarsys.sample.testutils

import android.content.Context

import androidx.multidex.MultiDex

import com.emarsys.sample.SampleApplication

class InstrumentedApplication : SampleApplication() {

    override fun attachBaseContext(base: Context) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}
